spec/serializers/provider_serializer_spec.rb

```diff
@@ -87,13 +87,10 @@
 
   describe 'contacts' do
     describe 'generate provider object returns the providers contacts' do
-      let(:provider) do
-        create :provider,
-               contacts: [contact]
-      end
-      let(:contact) { create(:contact) }
+      let(:contact)  { create :contact }
+      let(:provider) { create :provider, contacts: [contact] }
 
-      subject { serialize(provider)['contacts'].find { |c| c[:type] == contact.type } }
+      subject { serialize(provider)['contacts'].first }
```

Comment:
ain't this an admin?

Reply:
Is it? This test isn't about admin and we should be able to change the default type in the factory without affecting this test.

If anything, we could get rid of this test, but then we might be getting rid of the `admin` test below at some point (when we no longer have the fallback behaviour) and not have any testing for contact serialisation in the provider serialiser.

In the end, I just didn't think it was worth worrying too much about. What do you think we should do?